standalone/tests/test_scenario.py

```diff
@@ -0,0 +1,55 @@
+import time
+from uuid import uuid4
+
+from powersimdata import Scenario
+
+
+def create_default():
+    scenario = Scenario()
+    scenario.set_grid(interconnect="Texas")
+    scenario.set_name("test", "comp_" + str(uuid4()))
+    scenario.set_time("2016-01-01 00:00:00", "2016-01-01 03:00:00", "1H")
```

Comment:
I would suggest to change 1H to at least 2H so that it gives reasonable results for storage operations. See the conversation [here](https://github.com/Breakthrough-Energy/PowerSimData/issues/684) for details.